### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by ZennLuu**
was made:
 Test comunication programs for client and server.

### CODE REVIEW

**Comment by andrii-s-spitzer-lpnu-ua** [comment]
Can not comment in code. So let's comunicate here.
1) task was only readme and repo. It is very dificult to revew ALL CODE at ones. So please - split to daifferent pull requests per tasks.
2) remove all binaries from source control.

**Comment by ZennLuu** [comment]
README file may confused this pr as for task1, so i cleaned README to make it appropriate for task2.
Also removed binaries.

File: client/TTTClient/TTTClient/packages.config [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (andrii-s-spitzer-lpnu-ua) [inline client/TTTClient/TTTClient/packages.config:1]: Remove packages from git control. They should download while compile or build process.

Comment (ZennLuu) [reply client/TTTClient/TTTClient/packages.config:1]: Done. Now i understand purpose of this config file. Thanks.

File: server/server.ino [hunk 2]
```diff
@@ -0,0 +1,19 @@
+void setup() {
+  // Start the UART communication
+  Serial.begin(115200);
```
Comment (andrii-s-spitzer-lpnu-ua) [inline server/server.ino]: do nlt use magic numbers - create a const with meaningful name

File: server/server.ino [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  // Check if data is available to read
+  if (Serial.available()) {
+    // Read incoming message from client
+    String received_message = Serial.readStringUntil('\n');
+
+    String modified_message = received_message + " - Hello from ESP32!";
+
+    // Send the modified message back to the client
+    Serial.println(modified_message);
+    delay(1000); // Optional delay for stability
```
Comment (andrii-s-spitzer-lpnu-ua) [inline server/server.ino]: the same - magic namber

File: client/TTTClient/TTTClient/Form1.cs [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+namespace TTTClient
+{
+    public partial class Form1 : Form
+    {
+        public Form1()
+        {
+            InitializeComponent();
+
+            // For now: requires manual COM-port selection
+            SerialPort serial = new SerialPort("COM10", 115200);
```
Comment (andrii-s-spitzer-lpnu-ua) [inline client/TTTClient/TTTClient/Form1.cs]: the same comment about magic nuber as in server.ino. And do not pass "COM10" as string literal or const - each time you will require to change it whrn device connetcted to diferent port or other laptop is used. 
You can pass it as CLI argument in CLI app or add textbox or combobox on GUI. Combobox is more userfriendly solution but you need to get active COM ports list to show.

**Review by andrii-s-spitzer-lpnu-ua — APPROVED** [review]

File: client/TTTClient/TTTClient/Form1.cs [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+        public Form1()
+        {
+            InitializeComponent();
+
+            // Fill comboBox1 on initialization
+            fill_comboBox1();
+        }
+
+        // Send and recieve message if COM-Port selected
+        private void comboBox1_SelectedIndexChanged(object sender, EventArgs e)
```
Comment (andrii-s-spitzer-lpnu-ua) [inline client/TTTClient/TTTClient/Form1.cs:27]: approved, but combobox1 is not good name)

**Pull request merged**